core/server/data/migrations/versions/5.0/2022-04-21-15-44-backfill-mrr-events-for-canceled-subscriptions.js

```diff
@@ -0,0 +1,66 @@
+const ObjectID = require('bson-objectid').default;
+const logging = require('@tryghost/logging');
+
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const subscriptions = await knex('members_stripe_customers_subscriptions')
+            .leftJoin('members_paid_subscription_events', 'members_stripe_customers_subscriptions.id', '=', 'members_paid_subscription_events.subscription_id')
+            .join('members_stripe_customers', 'members_stripe_customers_subscriptions.customer_id', '=', 'members_stripe_customers.customer_id')
+            .join('members', 'members_stripe_customers.member_id', '=', 'members.id')
+            .where('members_stripe_customers_subscriptions.cancel_at_period_end', '=', true)
+            .where('members_stripe_customers_subscriptions.mrr', '!=', 0)
+            .where(knex.raw(`(members_paid_subscription_events.type = 'canceled' OR members_paid_subscription_events.type is null)`))
```

Comment:
any reason not to use the `.where((query) => {...})` syntax for this?

Reply:
Not particularly! Do you want me to change it?